pkg/wiparser/wiparser_test.go

```diff
@@ -95,47 +95,6 @@ func TestParse(t *testing.T) {
 			},
 			expectedErr: false,
 		},
-		{
-			name:  "valid dns and ip addresses with associated nodes",
-			input: map[string]string{"localhost": "username=core", "127.0.0.1": "username=Admin"},
```

Comment:
Instead of deleting this, how about modifying it to a valid case? i.e. change the second entry `"111.1.1.1"` (and associated changes to node and expectedOut)

Reply:
Thanks for catching that, you're right. I'm not sure why I wrote that off in the first place :facepalm:

Reply:
+1

Reply:
+ 1 for above comment